src/core/server/http/lifecycle/on_pre_response.ts

```diff
@@ -64,7 +64,7 @@ const preResponseResult = {
 };
 
 /**
- * A tool set defining an outcome of OnPreAuth interceptor for incoming request.
+ * A tool set defining an outcome of OnPreRouting interceptor for incoming request.
```

Comment:
__nit:__ looks like it should be `onPreResponse`?

Reply:
```suggestion
 * A tool set defining an outcome of OnPreResponse interceptor for incoming request.
```